index.bs

```diff
@@ -355,6 +371,53 @@ or failure:
 	append a duplicate of the second element to |tokens|.
 7. Return |tokens|.
 
+The <dfn for="IntersectionObserver">scrollable intersection rectangles</dfn>
+for an {{IntersectionObserver}}
+are the rectangles of scrollable containers on the path from the
+<a>intersection root</a> to <a for="IntersectionObserver">target</a>
+which could clip the target's rectangle.
+
+When calculating the <a>scrollable intersection rectangles</a> for
+a <a>same-origin-domain target</a>, the rectangle is then expanded
+according to the offsets in the {{IntersectionObserver}}’s {{[[scrollMargin]]}} slot
+in a manner similar to CSS's 'margin' property,
+with the four values indicating the amount the top, right, bottom, and left edges, respectively, are offset by,
+with positive lengths indicating an outward offset.
+Percentages are resolved relative to the width of the undilated rectangle.
+
+Note: {{IntersectionObserver/scrollMargin}} affects the clipping of <a for="IntersectionObserver">target</a>
+by all scrollable ancestors up to and including the <a>intersection root</a>.
+
+Note: <a>scrollable intersection rectangles</a> are not affected by
+<a>pinch zoom</a> and will report the unadjusted <a>viewport</a>, consistent with the
+intent of pinch zooming (to act like a magnifying glass and NOT change layout.)
+
+To <dfn for="IntersectionObserver">parse a scroll margin</dfn>
```

Comment:
If this text is identical to `parse a root margin`, can we combine them into a single `parse a root or scroll margin` section, or maybe `parse a margin option`?

Reply:
They are not quite identical. Scroll margin can also be an empty list.

Reply:
Updated scrollMargin to match rootMargin